CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Contributing
+
+This document outlines the ways to contribute to `python-dateutil`. This is a fairly small, low-traffic project, so most of the contribution norms (coding style, acceptance criteria) have been developed ad hoc and this document will not be exhaustive. If you are interested in contributing code or documentation, please take a moment to at least review the license section to understand how your code will be licensed.
+
+## Types of contribution
+### Bug reports
+Bug reports are an important type of contribution - it's important to get feedback about how the library is failing, and there's no better way to do that than to hear about real-life failure cases. A good bug report will include:
+
+1. A minimal, reproducible example - a small, self-contained script that can reproduce the behavior is the best way to get your bug fixed. For more information and tips on how to structure these, read [StackOverflow's guide to creating a minimal, complete, verified example](https://stackoverflow.com/help/mcve).
+
+2. The platform and versions of everything involved, at a minimum please include operating system, `python` version and `dateutil` version. Instructions on getting your versions:
+    - `dateutil`: `python -c 'import dateutil; print(dateutil.__version__)'`
+    - `Python`: `python --version`
+
+3. A description of the problem - what *is* happening and what *should* happen.
+
+While pull requests fixing bugs are accepted, they are *not* required - the bug report in itself is a great contribution.
```

Comment:
Do you accept xfailing test cases as bug reports?

Reply:
I haven't been a huge fan of that in the past, but it's a good point. I think it should be added when #519 is fixed.